Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -568,7 +603,16 @@ int main(int argc, char *argv[])
         return 0;
     }
 
-    auto backend = createViewBackend(1280, 720);
+    if (!parseWindowSize()) {
+#if ENABLE_WPE_PLATFORM
+        if (useWPEPlatformAPI) {
+            windowWidth = 1024;
+            windowHeight = 768;
+        }
```

Comment:
I understand now the comment on the initialization. I still think it's better to initialize to 0 and only resize if they are set.